tst/EnergyPlus/unit/ZoneEquipmentManager.unit.cc

```diff
@@ -62,11 +64,13 @@
 #include <EnergyPlus/DataSizing.hh>
 #include <EnergyPlus/DataZoneEnergyDemands.hh>
 #include <EnergyPlus/DataZoneEquipment.hh>
+#include <EnergyPlus/EarthTube.hh>
 #include <EnergyPlus/HeatBalanceAirManager.hh>
 #include <EnergyPlus/HeatBalanceManager.hh>
 #include <EnergyPlus/IOFiles.hh>
 #include <EnergyPlus/ScheduleManager.hh>
 #include <EnergyPlus/SimAirServingZones.hh>
+#include <EnergyPlus/ThermalChimney.hh>
```

Comment:
I can see the need for the new .hh files inclusion here as they are need for the Manges    ManageEarthTube(), ManageCoolTower(), and ManageThermalChimney() calls.